src/service/network.ts

```diff
@@ -70,7 +71,8 @@ export default function getNetwork(network: NetworkName): {
         'https://api.thegraph.com/subgraphs/name/ensdomains/ensgoerli';
       break;
     case NETWORK.MAINNET:
-      SUBGRAPH_URL = 'https://api.thegraph.com/subgraphs/name/ensdomains/ens';
+      SUBGRAPH_URL = 
+        `https://gateway-arbitrum.network.thegraph.com/api/${THE_GRAPH_API_KEY}/subgraphs/id/5XqPmWe6gjyrJtFn9cLy237i4cWw2j9HcUJEXsP5qGtH`;
```

Comment:
asking for curiosity but, is this address correct? under it's subname it refers arbitrum, it confused me.